packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -383,7 +411,11 @@ const Editor: FunctionComponent<EditorProps> = (props) => {
 
                 {withFloatingAddMenu && (
                     <FloatingAddMenu
-                        {...withFloatingAddMenu}
+                        tooltip={
+                            typeof withFloatingAddMenu === 'object'
```

Comment:
nit: here will be runtime error if for some reason withFloatingAddMenu will be null